cleaner/.jest/setEnvVars.js

```diff
@@ -0,0 +1,2 @@
+process.env.MAX_INACTIVE_DURATION = '2d';
+process.env.IS_DELETE = true;
```

Comment:
Can you rename this to something more grammatically appealing? Looks good to me otherwise!
```suggestion
process.env.SHOULD_DELETE = true;
```

Reply:
Done @bendehaan, thanks for your review